cmd/plume/prune.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2016 CoreOS, Inc.
```

Comment:
I think we can remove this line or change it

Reply:
Indeed. Sorry about that, copy & paste mistake.